usdt-tests-common/src/lib.rs

```diff
@@ -1,11 +1,10 @@
-use std::env;
 use std::process::Command;
 
 #[cfg(target_os = "illumos")]
 pub fn root_command(name: &str) -> Command {
     // On illumos systems, we prefer pfexec(1) but allow some other command to
     // be specified through the environment.
-    let pfexec = env::var("PFEXEC").unwrap_or_else(|_| "/usr/bin/pfexec".to_string());
+    let pfexec = std::env::var("PFEXEC").unwrap_or_else(|_| "/usr/bin/pfexec".to_string());
```

Comment:
So that I know in future: is there some rule/heuristic you're using to determine whether to import a module or merely fully qualify the call site?